### DISCUSSION TITLE
Error :  ElasticSearch (OpenDistro SQL)

### DISCUSSION THREAD

**Discussion by lazpuzzle** [apache/superset#19282] [Q&A / Help]
I got Error
ElasticSearch (OpenDistro SQL) Error
odelasticsearch error: TransportError(500, 'SearchPhaseExecutionException')

When i user sqllab and query is 
select * from table 
and set limit is 10000
or
select * from table limit more than 7000
how can i solve it

![image](https://user-images.githubusercontent.com/40905829/159245454-289c43ef-1847-4c45-98f0-c02945d64425.png)

**Comment by srinify** [comment]
Hi there, I think this may be an issue with your setup / config not a bug in Superset per say! I'm converting this to a discussion so someone can try to help you out!
